src/biokbase/narrative/tests/util.py

```diff
@@ -296,5 +296,43 @@ def find_free_port() -> int:
         return s.getsockname()[1]
 
 
+def validate_job_state(state: dict) -> None:
+    """
+    Validates the structure and entries in a job state as returned by the JobManager.
+    If any keys are missing, or extra keys exist, or values are weird, then this
+    raises an AssertionError.
+    """
+    # list of tuples - first = key name, second = value type
+    # details for other cases comes later. This is just the expected basic set of
+    # keys for EVERY job, once it's been created in EE2.
```

Comment:
So jobs launched not in narrative will not have run_id or cell_id, not sure if this matters here.

Reply:
That's a good point! I forgot about that. It shouldn't be forced, then, yes.

Reply:
I'll address that in #1601